code/game/objects/structures/wizard_wardrobe.dm

```diff
@@ -0,0 +1,84 @@
+/obj/structure/wizard_wardrobe
+	name = "mystical wardrobe"
+	desc = "An elegantly-carved closet made of enchanted wood. It stores copious amounts of robes, hats, and sandals."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "wizard_wardrobe"
+	density = 1
+	anchored = 1
+
+/obj/structure/wizard_wardrobe/attack_hand(mob/user as mob)
+	if(!user.canUseTopic(src))
+		return
+	if(ishuman(user))
+		var/mob/living/carbon/human/M = user
+		var/choice = input(user, "What do you want to take from the wardrobe?", "Clothes") as null|anything in list("Robes", "Hat", "Sandals")
+
+		if(!user.canUseTopic(src))
+			return
+		switch(choice)
+			if("Robes")
+				var/robe_type = null
+				var/robe_color = input(user, "What variety?", "Clothes") as null|anything in list("Blue", "Red", "Magus Blue", "Magus Red", "Yellow", "Witch")
+				switch(robe_color)
+					if("Blue")
+						robe_type = /obj/item/clothing/suit/wizrobe
+					if("Red")
+						robe_type = /obj/item/clothing/suit/wizrobe/red
+					if("Yellow")
+						robe_type = /obj/item/clothing/suit/wizrobe/yellow
+					if("Magus Blue")
+						robe_type = /obj/item/clothing/suit/wizrobe/magusblue
+					if("Magus Red")
+						robe_type = /obj/item/clothing/suit/wizrobe/magusred
+					if("Yellow")
+						robe_type = /obj/item/clothing/suit/wizrobe/yellow
```

Comment:
What the _fuck_. I fixed that at least three times. Thanks for the heads-up, I'll fix it.